src/core/components/app.jsx

```diff
@@ -10,6 +14,14 @@ export default class App extends React.Component {
     return Component ? Component : ()=> <h1> No layout defined for &quot;{layoutName}&quot; </h1>
   }
 
+  componentDidMount() {
+    makeDeeplinks(document.body)
+    //Focus on a deeplink's target whenever a deeplink is clicked
+    document.body.addEventListener("deeplinkClick", function() {
```

Comment:
Instead of listening to a custom global event, consider listening to the [`hashchange` event](https://developer.mozilla.org/en-US/docs/Web/API/WindowEventHandlers/onhashchange) which is a more idiomatic way to detect when a link has modified the URL fragment.